data/sql/updates/pending_db_world/rev_1646834195276318302.sql

```diff
@@ -0,0 +1,287 @@
+INSERT INTO `quest_request_items_locale` (`ID`, `locale`, `CompletionText`, `VerifiedBuild`) VALUES
+(11001, 'ruRU', 'Какие новости из Сетеккских залов? Ты $Gпобедил:победила; Анзу?', 0),
+(11005, 'ruRU', 'Дело сделано, $N? Я бы сам убил этих проклятых жрецов когтя, если бы не мои травмы.', 0),
+(11006, 'ruRU', 'Похоже, командир все же не бредил. Принесите мне еще пыли, и я приготовлю вам другой эликсир.', 0),
+(11007, 'ruRU', 'Ты $Gуслышал:услышала; мой зов...', 0),
+(11016, 'ruRU', 'Нам нужны эти шкуры на подкладку для брони и прочие технические нужды.', 0),
+(11023, 'ruRU', 'Ну, как бомбардировка?', 0),
+(11026, 'ruRU', 'Мы так думаем, ты $Gиспользовал:использовала; кристалл изгнания, чтобы послать целую кучу этих демонов Пылающего Легиона паковать вещички?!', 0),
+(11027, 'ruRU', 'Демоны из этого лагеря постоянно ремонтируют транспортера.$B$BГахку нужна темная руна, чтобы сделать выкованную кристалл с темными рунами.$B$BЕсли у маленького $R нет темной руны, поговори с Кронком. Он делает одну темную руну каждый день!', 0),
+(11029, 'ruRU', 'Книга, $N. Ты $Gдостал:достала; её?', 0),
+(11030, 'ruRU', 'Ты уже $Gдобыл:добыла; этот магический настой? Для Торкуса-младшего он имеет большое значение.', 0),
+(11036, 'ruRU', 'Это поставка от старины Орока?', 0),
+(11051, 'ruRU', '$GТакой:Такая; $Gопытный:опытная; $Gпрофессионал:профессионалка; с легкостью сможет изгнать демонов. Правда же, $R?', 0),
+(11056, 'ruRU', 'Ты вернулся, $N. Я уже начал думать, что ты меня бросил.', 0),
+(11058, 'ruRU', 'Вы уже научились пользоваться апекситовой реликвией? И нам, и Ша'тарским Небесным Стражам к северу интересно, что вы о них узнали.', 0),
```

Comment:
The problem  now is, that you have to escape some of the single qutes like this `\'` or `''`